AIRS_CLDDET.NL

```diff
@@ -0,0 +1,21 @@
+&Cloud_Detect_Coeffs
+N__Band_Size(1)=138
+N__Bands(1:138,1)=  1,   6,   7,  10,  11,  15,  16,  17,  20,  21,
+                   22,  24,  27,  28,  30,  36,  39,  40,  42,  51,
+                   52,  54,  55,  56,  59,  62,  63,  68,  69,  71,
+                   72,  73,  74,  75,  76,  77,  78,  79,  80,  82,
+                   83,  84,  86,  92,  93,  98,  99, 101, 104, 105,
+                  108, 110, 111, 113, 116, 117, 123, 124, 128, 129,
+                  138, 139, 144, 145, 150, 151, 156, 157, 159, 162,
+                  165, 168, 169, 170, 172, 173, 174, 175, 177, 179,
+                  180, 182, 185, 186, 190, 192, 193, 198, 201, 204,
+                  207, 210, 213, 215, 216, 218, 221, 224, 226, 227,
+                  232, 239, 248, 250, 251, 252, 253, 256, 257, 261,
+                  262, 267, 272, 295, 299, 305, 308, 309, 310,
+                  318, 321, 333, 338, 355, 362, 375, 475,
+                  484, 497, 528, 587, 672, 787, 791, 843, 870, 914,
```

Comment:
Minor nitpick: is there are reason for not extending these lines out to 10 bands each?